### PR TITLE
Feature/reorganizing child dashboard

### PR DESCRIPTION
**Pull request by artofmayhem**
This PR adjusts and reorganizes the child dashboard to expected initial functionality. 

Join button - replaced by link to Trophy Room. This logic will live in the choosing of game mode feature. and will be conditionally rendered as accept the mission or Join (re-join) the mission 
Accept Mission button - no changes
leaderboard button - moved from bottom right to bottom left
Trophy Room - inserted bottom right and styled

### CODE REVIEW

File: src/components/pages/ChildDashboard/RenderChildDashboard.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { useHistory } from 'react-router-dom';
 import { InstructionsModal } from '../../common';
 import { modalInstructions } from '../../../utils/helpers';
 import { HUD } from '../HeadsUpDisplay/index';
-
-import adventure_passport from '../../../assets/images/child_dashboard_images/adventure_passport.svg';
+import Explosion from '../../../assets/images/gamemodeimg/explosion.png';
+// import adventure_passport from '../../../assets/images/child_dashboard_images/adventure_passport.svg';
```
Comment (ghost) [inline src/components/pages/ChildDashboard/RenderChildDashboard.js]: This import should be deleted instead of commented out.

**Review by ghost — CHANGES_REQUESTED** [review]
Good job getting all the cards to go to the right place and setting everything up.

File: src/components/pages/ChildDashboard/RenderChildDashboard.js (hunk 1 above)
Comment (artofmayhem) [reply src/components/pages/ChildDashboard/RenderChildDashboard.js]: done

**Review by ghost — APPROVED** [review]

**Review by rhiannonstanford — APPROVED** [review]
Nice work. Tested and functions well.

**Pull request merged**